packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/GroupForm.tsx

```diff
@@ -0,0 +1,477 @@
+import React, { useEffect, useState } from 'react';
+import app from 'state';
+import { useFetchTopicsQuery } from 'state/api/topics';
+import { CWDivider } from 'views/components/component_kit/cw_divider';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWTextArea } from 'views/components/component_kit/cw_text_area';
+import { CWForm } from 'views/components/component_kit/new_designs/CWForm';
+import { CWSelectList } from 'views/components/component_kit/new_designs/CWSelectList';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import { MessageRow } from 'views/components/component_kit/new_designs/CWTextInput/MessageRow';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import { CWRadioButton } from 'views/components/component_kit/new_designs/cw_radio_button';
+import { ZodError } from 'zod';
+import TopicGatingHelpMessage from '../../TopicGatingHelpMessage';
+import './GroupForm.scss';
+import RequirementSubForm from './RequirementSubForm';
+import {
+  GroupFormProps,
+  RequirementSubType,
+  RequirementSubTypeWithLabel,
+} from './index.types';
+import {
+  VALIDATION_MESSAGES,
+  groupValidationSchema,
+  requirementSubFormValidationSchema,
+} from './validations';
+
+type CWRequirementsRadioButtonProps = {
+  inputError?: string;
+  inputValue: string;
+  onInputValueChange: (value: string) => any;
+};
+
+const CWRequirementsRadioButton = ({
```

Comment:
1. replied [here](https://github.com/hicommonwealth/commonwealth/pull/5045#discussion_r1344108739)
2. yes its only used in this place and is basically baked in (non-generic).

Reply:
I am not gonna argue here because I am also not sure when we should and when we should not use CW prefix before the component name. Maybe @gdjohnson would know or document that if possible?